test/server/cards/02_SHD/units/TheMandalorian.spec.ts

```diff
@@ -0,0 +1,42 @@
+describe('The Mandalorian, Wherever I Go He Goes', function () {
+    integration(function (contextRef) {
+        describe('The Mandalorian\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['the-mandalorian#wherever-i-go-he-goes'],
+                        groundArena: [ 
+                            { card: 'grogu#irresistible', damage: 3 },
+                            { card: 'mandalorian-warrior' }
```

Comment:
you can omit {card:} and write directly 'mandalorian-warrior' if there is not properties on unit